### ISSUE TITLE
Make video thumbnail 16:9 ratio

### ISSUE THREAD

**Issue by AkshatAdsule** [abhisheknaiidu/A-POP#5]
Thumbnails on YouTube are at a 16:9 ratio, but it is a different ratio on the website. Making it 16:9 would allow A-POP to show the whole thumbnail, and not cut anything off.

**Comment by abhisheknaiidu** [comment]
@AkshatAdsule thanks for opening the issue, 
yes it actually makes sense, will you interested to work on it?

**Comment by AkshatAdsule** [comment]
Yeah for sure! I haven't looked at your code yet, I just saw your repo while exploring GitHub, but I'd be happy to help in anyway.

**Comment by abhisheknaiidu** [comment]
ok @AkshatAdsule I'm assigning you then!

**Comment by AkshatAdsule** [comment]
#7 Fixed!
